pages/phases/job-analysis/defining-proficiency-levels.md

```diff
@@ -6,48 +6,91 @@ title: Defining Proficiency Levels and Required Experience
 
-#### Example Competency: Collaboration
+**Competent:** Applies the competency in difficult situations, requiring occasional guidance. Able to successfully complete tasks in this competency as requested. Demonstrates understanding of terminology, concepts, principles, and issues related to this competency.
+ 
+> Competant performers can see their actions in terms of long-range goals or plans. They are consciously aware of formulating, evaluating, and modifying goals-plans. The competent performer is able to generate plans in terms of current and contemplated future aspects that are most important, and those that are not. The competent performer lacks the speed and flexibility that emerges at higher levels of expertise but has a sense of mastery and the ability to cope with and manage a variety of situations.
 
-The ability to work cooperatively with others to accomplish objectives and build inclusive and mutually beneficial partnerships.
+**Advanced:** Applies the competency in considerably difficult situations, generally requiring little or no guidance. Recognized within immediate organization as "a person to ask" when difficult questions arise regarding this skill. Demonstrates broad understanding of terminology, concepts, principles, and issues related to this competency.
 
-* **Unfamiliar:** Has not worked in cross-functional teams or collaborated closely with people from other disciplines or worked in teams that encourage diverse perspectives.
-* **Familiar:** Has experience constructively discussing and integrating design feedback from peers, clients, or stakeholders.
-* **Experienced (Required):** Demonstrates success in working with large organizations, highly regulated domains, or highly bureaucratic environments.
-* **Master:** Leads projects with changing priorities or competing interests. Is skilled at negotiating or mediating conflicts. Coordinates across group boundaries to achieve organizational as well as design goals.
+> Advanced performers perceive situations as wholes, rather than in terms of situational components. Their performance is guided by "maxims." Perception is key. The perspective is NOT thought out but "presents itself" based upon experience. The proficient performer has learned what typical events to expect in a given situation and how plans need to be modified in accord with these events. This also means that she or he can recognize when the expected typical picture does not materialize and can modify plans and goals accordingly. Situational aspects stand out as more or less important in *this* situation.
```

Comment:
```suggestion
> Advanced performers perceive situations as wholes, rather than in terms of situational components. Their performance is guided by "maxims." Perception is key. Perspectives on common problems do not require significant time spent thinking things through, rather the solution "presents itself" based upon accumulated experience. The proficient performer has learned what typical events to expect in a given situation and how plans need to be modified in accord with these events. This also means that they can recognize when the expected typical picture does not materialize and can modify plans and goals accordingly. Situational aspects stand out as more or less important in *this* situation.
```

Reply:
@wslack -- Generally speaking, use 3rd person gender-neutral "they" whenever possible, avoiding deciding whether to put "he" or "she" first, and including non-binary gender identities.

Reply:
Yep, I should have caught that beyond the pronoun order. Thank you.